src/components/safe-apps/SafeAppsSection/styles.module.css

```diff
@@ -0,0 +1,24 @@
+.accordion {
+    box-shadow: none;
```

Comment:
Looks like 4 spaces. Needs to be 2.

Reply:
Hmm, it seems like prettier is not enforced for `.css`, nothing happens when I do the formatting. I'll research how to apply the rules to `css` files

Reply:
Please see https://github.com/safe-global/web-core/pull/443